### DISCUSSION TITLE
软件界面简洁一下如何？

### DISCUSSION THREAD

**Discussion by a18ccms** [localsend/localsend#1131] [L-简体中文]
我提议软件界面简洁一下，左侧三个分栏感觉用处不是很大，主要界面还是发送那里，另外两个分栏完全可以合成两三个按钮。这样软件一打开就是发送界面。省着还得点击发送。大家觉得如何？

**Comment by Jacen918** [comment]
认同，我也觉得一个页面就够了，而且最好是做成像微信、QQ这种会话窗口式，将历史记录，以会话形式展示在窗口。把发送文件，就像发微信一样，这样同事之前还可以用来沟通聊工作更方便。
如果有需要我还可以帮忙设计UI
或者有哪个大佬来个二开吧
